code/game/objects/items/random_items.dm

```diff
@@ -130,8 +130,7 @@
 
 /obj/item/storage/pill_bottle/random_meds/New()
 	..()
-	var/i = 1
-	while(i < storage_slots)
+	for(var/i=1, i<storage_slots, i++)						// Why was this a while loop?
```

Comment:
use the proper format, please.

Reply:
what do you mean by proper format?

Reply:
`for(var/i in 1 to X)`

Reply:
[This contributing guide](https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md) has a helpful section describing the 'most properer' format and why it's used. BYOND seems to favour it over all other styles of for loop